src/hooks/useLastApplicationDeployTime.ts

```diff
@@ -0,0 +1,23 @@
+import { useK8sWatchResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { ComponentGroupVersionKind } from '../models';
+import { ComponentKind } from '../types';
+
+export const useLastApplicationDeployTime = (
```

Comment:
Not ideal to be using component creation timestamp. However there's issues still with getting an accurate last deploy timestamp.
See https://issues.redhat.com/browse/HAC-2207